uint8 id
---
int32 min_position
int32 max_position